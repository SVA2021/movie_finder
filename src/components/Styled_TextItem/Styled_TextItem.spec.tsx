import { render, screen } from '@testing-library/react';
import Item from './Styled_TextItem';


describe('render styled-text-item', () => {

	const setUp = () => render(
		<Item>Hello World</Item>
	);

	it('snapshot test', () => {
		setUp();
		expect(screen.getByTestId('styled-text-item')).toMatchSnapshot();
	});

	it('find text test', () => {
		setUp();
		expect(screen.getByText(/hello world/i)).toBeInTheDocument();
	});

	it('not find text test', () => {
		setUp();
		expect(screen.queryByText(/another text/i)).toBeNull();
	});

	screen.debug();

})