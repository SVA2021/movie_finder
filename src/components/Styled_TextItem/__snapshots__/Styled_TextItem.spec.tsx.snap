// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render styled-text-item snapshot test 1`] = `
<p
  class="MuiTypography-root MuiTypography-h3 css-53zoc3-MuiTypography-root"
  data-testid="styled-text-item"
>
  Hello World
</p>
`;
